etc/requirements.txt

```diff
@@ -9,3 +9,5 @@ networkx==1.11
 pytest>=3.0.5
 sphinx>=1.4.1
 plotly>=1.3.2
+python-igraph
```

Comment:
@hrishikesh-dahiya version numbers would be required for the newly added packages.

Reply:
Please go with the latest version that works with Python 3.5
I remember iGraph not being available for Python 3.6 at all